### DISCUSSION TITLE
Is there any plugin to show outline navigation bar in editing mode to locate any heading section easily

### DISCUSSION THREAD

**Discussion by cruvie** [Milkdown/milkdown#636] [Q&A]
like official document has the side outline navgation bar,but doesn't in editing demo online
![image](https://user-images.githubusercontent.com/55651847/180919619-f1a0c4e9-18d7-4933-83d6-e705900c513c.png)
![image](https://user-images.githubusercontent.com/55651847/180919693-c62f6084-7155-43ab-be08-fdfb6c3d0448.png)

**Comment by Saul-Mirone** [comment]
There's no official one. You can write a community plugin if you have interests in it.
